pkgs/applications/editors/kakoune/plugins/generated.nix

```diff
@@ -17,12 +17,12 @@ let
 
   auto-pairs-kak = buildKakounePluginFrom2Nix {
     pname = "auto-pairs-kak";
-    version = "2020-10-04";
+    version = "2021-03-28";
```

Comment:
Can you hold back the update of `auto-pairs-kak`?
The updated version needs `kakoune-cr`, which is not yet in nixpkgs (#115011).